modules/osdk-owned-crds.adoc

```diff
@@ -0,0 +1,135 @@
+[id="osdk-crds-owned_{context}"]
+== Owned CRDs
+
+The CRDs owned by your Operator are the most important part of your CSV. This
+establishes the link between your Operator and the required RBAC rules,
+dependency management, and other Kubernetes concepts.
+
+It is common for your Operator to use multiple CRDs to link together concepts,
+such as top-level database configuration in one object and a representation of
+ReplicaSets in another. Each one should be listed out in the CSV file.
+
+.Owned CRD fields
+[cols="2a,5a,2",options="header"]
+|===
+|Field |Description |Required/Optional
+
+|`Name`
+|The full name of your CRD.
+|Required
+
+|`Version`
+|The version of that object API.
+|Required
+
+|`Kind`
+|The machine readable name of your CRD.
+|Required
+
+|`DisplayName`
+|A human readable version of your CRD name, for example `MongoDB Standalone`.
+|Required
+
+|`Description`
+|A short description of how this CRD is used by the Operator or a description of
+the functionality provided by the CRD.
+|Required
+
+|`Group`
+|The API group that this CRD belongs to, for example `database.example.com`.
+|Optional
+
+|`Resources`
+a|Your CRDs own one or more types of Kubernetes objects. These are listed in the resources section to inform your users of the objects they might need to troubleshoot or how to connect to the application, such as the Service or Ingress rule that exposes a database.
+
+It is recommended to only list out the objects that are important to a human,
+not an exhaustive list of everything you orchestrate. For example, ConfigMaps
+that store internal state that should not be modified by a user should not
+appear here.
+|Optional
+
+|`SpecDescriptors`, `StatusDescriptors`, and `ActionDescriptors`
+a|These Descriptors are a way to hint UIs with certain inputs or outputs of your
+Operator that are most important to an end user. If your CRD contains the name
+of a Secret or ConfigMap that the user must provide, you can specify that here.
+These items are linked and highlighted in compatible UIs.
+
+There are three types of descriptors:
+
+* `SpecDescriptors`: A reference to fields in the `spec` block of an object.
+* `StatusDescriptors`: A reference to fields in the `status` block of an object.
+* `ActionDescriptors`: A reference to actions that can be performed on an object.
+
+All Descriptors accept the following fields:
+
+* `DisplayName`: A human readable name for the Spec, Status, or Action.
+* `Description`: A short description of the Spec, Status, or Action and how it is
+used by the Operator.
+* `Path`: A dot-delimited path of the field on the object that this descriptor describes.
+* `X-Descriptors`: Used to determine which "capabilities" this descriptor has and
+which UI component to use. See the *openshift/console* project for a canonical
+link:https://github.com/openshift/console/blob/master/frontend/public/components/operator-lifecycle-manager/descriptors/types.ts#L5-L26[list of React UI X-Descriptors] for {product-title}.
```

Comment:
This link has been removed. @ecordell Could you help paste the correct link here? Thanks!

Reply:
@jianzhangbjz @ecordell How about https://github.com/openshift/console/tree/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors ?

Reply:
Or maybe better to match the release branch: https://github.com/openshift/console/tree/release-4.3/frontend/packages/operator-lifecycle-manager/src/components/descriptors

But either way, I can just get this ported into the product docs proper.

Reply:
Thanks!